server/controllers/PartyController.js

```diff
@@ -0,0 +1,32 @@
+
+import PartyModel from '../models/PartyModel';
+
+const { createParty } = PartyModel;
+
+/**
+ *Defines the actions for Party Endpoints
+ *@class PartyController
+ */
+class PartyController {
+  /**
+   *
+   * @param {object} req - request
+   * @param {object} res - response
+   */
+  static async createParty(req, res) {
+    let party;
+    try {
+      party = await createParty(req, res);
```

Comment:
I'm not sure why you are making use of the await keyword in front of createParty, I see that this function was implemented with the await keyword which has already returned the result

Reply:
The previous function returns a promise, so for me to be to read the value returned, I have to await the response too, that's the reason I had to use async/await. For instance this part `party = await createParty(req, res);` return *Promise <pending*>. So I have to await the Promise to resolve.